frontend/src/metabase/components/ErrorCard.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from "react";
+import PropTypes from "prop-types";
+import BodyComponent from "metabase/components/BodyComponent";
+import Card from "metabase/components/Card";
+import Icon from "metabase/components/Icon";
+
+function ErrorCard({ errorInfo }) {
+  const [showError, setShowError] = useState(false);
+
+  useEffect(() => {
+    if (errorInfo) {
+      setShowError(true);
+    }
+  }, [errorInfo]);
+
+  return showError ? (
+    <Card className="fixed right bottom zF" p={2} m={2} width={350}>
+      <div className="flex justify-between align-center mb1">
+        <div className="text-error flex align-center">
+          <Icon name="info_outline" mr={1} size="20" />
+          <h2>Something went wrong</h2>
```

Comment:
Shouldn't we translate this? That is - I believe there's already a translation for this phrase. Just wrap it in `t`.

Reply:
Good catch -- fixed